### PR TITLE
fix(zk): fix bug on vkey generation (part 1)

### PR DESCRIPTION
**Pull request by chokobole**
# Description

VerifyingKey generation contains so many bugs.. since this has been not tested. (It hasn't even been compiled for a while). This PR contains fixes some part of bug for a quick and easy review

### CODE REVIEW

File: tachyon/zk/plonk/circuit/floor_planner/simple_lookup_table_layouter.h [hunk 1]
```diff
@@ -19,6 +19,8 @@ namespace tachyon::zk {
 template <typename F>
 class SimpleLookupTableLayouter : public LookupTable<F>::Layouter {
  public:
+  using AssignCallback = typename LookupTable<F>::Layouter::AssignCallback;
+
```
Comment (TomTaehoonKim) [inline tachyon/zk/plonk/circuit/floor_planner/simple_lookup_table_layouter.h:23]: Do we need this type declaration? I can't find a reason in this PR.

**Review by TomTaehoonKim — CHANGES_REQUESTED** [review]

File: tachyon/zk/plonk/circuit/assembly.h [hunk 2]
```diff
@@ -23,8 +23,8 @@ template <typename PCSTy>
 class Assembly : public Assignment<typename PCSTy::Field> {
  public:
   using F = typename PCSTy::Field;
-  using DensePoly =
-      math::UnivariateDensePolynomial<math::RationalField<F>, PCSTy::kMaxSize>;
+  using DensePoly = math::UnivariateDensePolynomial<math::RationalField<F>,
+                                                    PCSTy::kMaxDegree>;
```
Comment (dongchangYoo) [inline tachyon/zk/plonk/circuit/assembly.h:27]: A `fixeds_`'s type is evaluations of rational field (not a dense poly!)
So, `DensePoly` should be fixed into `Evals` or `RationalEvals` or ...?

**Review by TomTaehoonKim — APPROVED** [review]
LGTM

**Review by Insun35 — APPROVED** [review]
LGTM

**Review by dongchangYoo — APPROVED** [review]
LGTM

**Review by TomTaehoonKim — APPROVED** [review]
LGTM

**Pull request merged**